doc/nrf/protocols/protocol_serialization/index.rst

```diff
@@ -0,0 +1,280 @@
+
+   .. code-block:: console
+
+      uart:~$ [00:00:03.392,517] <dbg> NRF_RPC: nrf_rpc_init: Done initializing nRF RPC module
+
+
+   This indicates that the communication between the devices has been initialized properly.
+
+#. Run the following command on the client's console to bring the OpenThread interface on the server device up:
+
+   .. code-block:: console
+
+      uart:~$ ot ifconfig up
+      Done
+      [00:02:28.980,041] <dbg> NRF_RPC: cmd_ctx_alloc: Command context 0 allocated
+      [00:02:28.980,102] <dbg> NRF_RPC: nrf_rpc_cmd_common: Sending command 0x00 from group 0x01
+      [00:02:28.980,133] <dbg> nrf_rpc_uart: send: Sending frame
+                                             80 00 ff 01 01 f6
+      ...
+
+
+#. Bring the corresponding Zephyr network interface on the client device:
+
+   .. code-block:: console
+
+      uart:~$ net iface up 1
+      Interface 1 is up
+
+#. Verify that the Zephyr network interface has been automatically configured with OpenThread's link-local address:
+
+   .. code-block:: console
+
+      uart:~$ ot ipaddr
+      fe80:0:0:0:6c26:956a:813:1e33
+      Done
+      ...
+
+      uart:~$ net iface  show
+
+      Interface net0 (0x200012c8) (<unknown type>) [1]
+      =========================================
+      MTU       : 1280
+      Flags     : NO_AUTO_START,IPv6,NO_ND,NO_MLD
+      Device    : ot_rpc (0x2b748)
+      IPv6 unicast addresses (max 5):
+            fe80::6c26:956a:813:1e33 autoconf preferred infinite
+
+#. Start Thread and become a leader:
+
+   .. code-block:: console
+      uart:~$ ot thread start
+      Done
+      ...
+
+      uart:~$ ot state leader
+      Done
+
+#. Verify that the Zephyr network interface has automatically received OpenThread's mesh-local addresses:
+
+   .. code-block:: console
+
+      uart:~$ net iface  show
+
+      Interface net0 (0x200012c8) (<unknown type>) [1]
+      =========================================
+      MTU       : 1280
+      Flags     : NO_AUTO_START,IPv6,NO_ND,NO_MLD
+      Device    : ot_rpc (0x2b748)
+      IPv6 unicast addresses (max 5):
+            fe80::6c26:956a:813:1e33 autoconf preferred infinite
+            fdde:ad00:beef:0:e503:abfd:1c8d:2664 autoconf preferred infinite meshlocal
+            fdde:ad00:beef::ff:fe00:fc00 autoconf preferred infinite meshlocal
+
+
+   This happens because the client registers a notification callback for OpenThread state changes at the server device and it continuously refreshes the client's IPv6 address list when that changes on the server side.
+
+#. Retrieve the operational dataset of the Thread network:
+
+    .. code-block:: console
+
+      uart:~$ ot dataset active -x
+      0e080000000000000000000300000b35060004001fffe00208dead00beef00cafe0708fddead00beef00000510f7893f15a55d8adeacad288c38bf32cc030a4f70656e546872656164010240120410d48d777a474f80e61aa5680de764bd6d0c0402a0f7f8
+      Done
+
+#. To send a UDP packet to a peer device and UDP port 5555, run the following command:
```

Comment:
Just note. With enabled debug snippet, I wasn't able to pass the last two steps since IP address is overwritten by debug information on screen. Customer has to keep in mind IP address, that is inconvenient :)